index.d.ts

```diff
@@ -1781,5 +1776,53 @@ export namespace stream {
 			addWorkbookRels(): Promise<void>;
 			addWorkbook(): Promise<void>;
 		}
+
+
+		interface WorkbookStreamReaderOptions {
+			/**
+			 * @default 'emit'
+			 */
+			entries: 'emit' | void;
```

Comment:
I don't understand whats means default here? how do you determine if it's set or not?

Reply:
I've just pushed an update to that - does that make it clearer?

Reply:
yeah, that's perfect